README.md

```diff
@@ -1,10 +1,88 @@
 # API Client Generator for TypeScript
 
-**This is not an official Google product.**
+## Background
+
+This tool is a TypeScript client library generator for network APIs specified
+by [protocol
+buffers](https://developers.google.com/protocol-buffers/) (including, but 
+not limited to, [client libraries](https://www.npmjs.com/search?q=%40google-cloud) 
+for Google Cloud APIs).
+
+It can be used for any API that uses protocol buffers and follows the conventions 
+described in [API Improvement Proposals](https://aip.dev/).
+
+## Getting started
+
+The [Showcase API](https://github.com/googleapis/gapic-showcase) is a good API to 
+play with if you want to start generating your own client libraries. It has several
+services, we'll use `Echo` service as an example.
+
+### Proto definitions for Echo service
+
+Take a look at [echo.proto](https://github.com/googleapis/gapic-showcase/blob/master/schema/google/showcase/v1beta1/echo.proto) that describes a simple
+network service.
+
+Download the proto files locally (the following examples work in Linux or macOS):
+
+```sh
+$ curl -L https://github.com/googleapis/gapic-showcase/releases/download/v0.6.0/gapic-showcase-0.6.0-protos.tar.gz | tar xz
+```
+
+### Run the generator to generate TypeScript libraries
+
+The easiest way to get started is to use our Docker image:
+
+```sh
+$ mkdir showcase-typescript
+$ docker run --rm \
+  --mount type=bind,source=`pwd`/google/showcase/v1beta1,destination=/in/google/showcase/v1beta1,readonly \
+  --mount type=bind,source=`pwd`/showcase-typescript,destination=/out \
```

Comment:
should we give users the place to download the docker image? Or we will add it later, thanks!

Reply:
That's exactly the command that should be used, according to [AIP 4290](https://aip.dev/4290). There is no need to download Docker image, `docker run` will pull the image from gcr.io.